srcpkgs/drawterm-9front/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'drawterm-9front'
+pkgname=drawterm-9front
+version=r393.cdbacd80f61a
+revision=1
+hostmakedepends="mercurial"
+makedepends="libX11-devel libXt-devel"
+short_desc="Drawterm fork by 9front project"
+maintainer="Juan Cuzmar <juan.cuzmar.s@gmail.com>"
+license="LPL-1.02"
+homepage="http://drawterm.9front.org/"
+CFLAGS="-fno-strict-aliasing"
+
+if [ "${XBPS_TARGET_MACHINE}" = armv5tel ]; then
+	CFLAGS+=" -DARMv5"
+fi
+
+do_fetch() {
+	hg clone https://code.9front.org/hg/drawterm ${wrksrc}
+}
+
+do_configure() {
+	local _arch;
+
+	case "$XBPS_TARGET_MACHINE" in
+		arm*) _arch=arm ;;
+		arm64*) _arch=arm64 ;;
+		i686*) _arch=386 ;;
+		mips*) _arch=mips ;;
+		x86_64*) _arch=amd64 ;;
+		ppc*) _arch=power ;;
+	esac
+
+	sed -i Make* */Makefile \
+		-e 's%$(CFLAGS)%$(_CFLAGS)%' \
+		-e 's%$(LDFLAGS)%$(_LDFLAGS)%'
+	sed -i -e 's%^CFLAGS=%_CFLAGS=$(CFLAGS) %' \
+		-e 's%^LDFLAGS=%_LDFLAGS=$(LDFLAGS) %' \
+		-e "s%arch=.*%arch=$_arch; \\\\%" Make.unix
+}
+
+do_build() {
+	make AR=$AR RANLIB=$RANLIB CC=$CC CFLAGS="$CFLAGS" LDFLAGS="$LDFLAGS" CONF=unix ${makejobs}
+}
+do_install() {
+	vbin drawterm
+	vlicense LICENSE
+	vman drawterm.1
+}
```

Comment:
Put a newline between the two functions.

Any reason you can't use the `gnu-makefile` build style? The `do_configure` function above could be `pre_configure` instead. And then you need to define only `do_install`, if `make install` doesn't do what you need.

Reply:
I could try that build style. but the project uses bare makefiles. but if with gnu-makefile works i'll put it.